src/menu/menu.scss

```diff
@@ -85,7 +85,7 @@
   
       > .iui-icon {
         @include themed {
-          fill: t(iui-text-color-muted);
+          fill: t(iui-icons-color-actionable-disabled);
```

Comment:
It looks like the "grey states" are the ones that have changed the most. Would it be possible to reduce the number of visual changes in test images where not needed?

It's good to use icon-specific variables but we need to make sure that all visual changes are fully intentional and we are not just going with it because it's convenient.

Reply:
Would it be ok to have _some_ icons using the `iui-color-foreground-` variables then?  Because if not, I'm not sure how else to fix this.

Reply:
It's not necessarily a bad thing that all these images changed. I would just like you to make sure every single visual change is intentional, as you are the UX person, checking for things like contrast

Reply:
For me new disabled color looks better 😀